src/taglibs/migrate/all-tags/dynamic-attributes.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function migrate(el, context) {
+    el.forEachAttribute(attr => {
+        const name = attr.name;
+        const value = attr.value;
+        if (
+            name == undefined &&
```

Comment:
This should use the same check that is used [here](https://github.com/marko-js/marko/blob/7d37fe36347fc75deb4148e9196f57e51b13121c/src/taglibs/core/core-transformer.js#L134) (`!attr.name && !attr.spread`). The implementation here will fail if you have something like `...({ a : 1 })` or any other type of node that isn't and identifier or member expression.

Also the previous warning should be removed from [here](https://github.com/marko-js/marko/blob/7d37fe36347fc75deb4148e9196f57e51b13121c/src/taglibs/core/core-transformer.js#L134) as well.